guides/shipments/shipping.md

```diff
@@ -1,129 +1,194 @@
+The shipping address entered during checkout defines the zone the customer is
+in. This limit the available shipping methods and defines regional taxation
+rules.
 
-### Shipping Categories
+<!-- TODO:
+  For more information about zones, see [the Locations guide](../locations).
+-->
 
-Shipping Categories are useful if you sell products whose shipping pricing vary depending on the type of product (TVs and Mugs, for instance) or the handling of the product (frigile or large). Shipping categories can be assigned when editing a product.
+### Shipping calculators
 
-**Note:** *For simple setups, where shipping for all products is priced the same (ie. T-shirt-only shop), all products would be assigned to the default shipping category for the store.*
+A shipping calculator is the component responsible for calculating the shipping
+rate for each available shipping method.
 
-Some examples of Shipping Categories would be:
+Solidus ships with five default shipping calculators:
 
-* Light (for lightweight items like stickers)
-* Regular
-* Heavy (for items over a certain weight)
+- [Flat percent](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/calculator/shipping/flat_percent_item_total.rb)
+- [Flat rate (per order)](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/calculator/shipping/flat_rate.rb)
+- [Flat rate per package item](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/calculator/shipping/per_item.rb)
+- [Flexible rate per package item](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/calculator/shipping/flexi_rate.rb)
+- [Price sack](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/calculator/shipping/price_sack.rb)
+  (which offers variable shipping cost that depends on order total)
 
-Shipping Categories are created in the admin interface (Settings -> Shipping -> Shipping Categories) and then assigned to products (Products -> Edit).
+Flexible rate is defined as a flat rate for the first product, plus a different
+flat rate for each additional product.
 
-During checkout, the shipping categories of the products in your order will determine which calculator will be used to price its shipping for each Shipping Method.
+If you want to estimate shipping rates using carrier APIs, You can use an
+extension like [`solidus_active_shipping`][solidus-active-shipping]. Or, if you
+have other complex needs, you can create a [custom shipping
+calculators][custom-shipping-calculators] for more information.
 
-### Shipping Calculators
+### Shipping rates
 
-A Calculator is the component responsible for calculating the shipping price for each available Shipping Method.
+For each shipment, a `Spree::ShippingRate` object is created for each of your
+store's shipping methods. These objects calculate how much the shipment would
```

Comment:
They represent the cost, and are calculated by the shipping method's calculator. Shipping rates don't change, they get deleted and re-created.